models/Order.js

```diff
@@ -7,25 +7,47 @@ const orderSchema = new mongoose.Schema({
   status: {
     type: String,
     required: true,
-    enum: ['sent', 'pending', 'canceled'],
-    default: 'pending',
+    enum: ['CREATED', 'CONFIRMED', 'CANCELLED', 'REFUNDED', 'SENT', 'DELIVERED'],
+    default: 'CREATED',
   },
   user: {
     firstName: String,
     lastName: String,
+    patronymicName: String,
     email: String,
     phoneNumber: Number,
-    address: Address,
   },
   dateOfCreation: {
     type: Date,
     default: Date.now,
   },
+  lastUpdatedDate: Date,
+  completed: Boolean,
+  userComment: {
+    type: String,
+    default: '',
+  },
+  adminComment: {
+    type: String,
+    default: '',
+  },
+  cancellationReason: {
+    type: String,
+    default: '',
+  },
   delivery: {
     sentOn: Date,
     sentBy: String,
+    courier: Boolean,
+    courierOffice: Number,
     invoiceNumber: String,
+    serviceType: {
+      type: String,
+      enum: ['WarehouseWarehouse', 'WarehouseDoors'],
+    },
+    cost: [CurrencySet],
   },
+  address: Address,
```

Comment:
Why have you moved `address` from the `user` object to the root order level?

Reply:
In case of a courier delivery, it stores client's address; in case of a delivery to carrier provider office, it stores address of the office